### DISCUSSION TITLE
New commits and v4.0

### DISCUSSION THREAD

**Discussion by sweetppro** [transmission/transmission#3240] [General]
It’s great to see all the new activity, I’ve been using transmission since macOS  leopard? maybe even earlier…

I’m looking forward to the official v4 update, and happy to help out with such a great project. Cheers everybody.

great work!

**Comment by Oleg-Chashko** [comment]
@sweetppro 
If you have the desire and time, please see my issues. Thank you.

**Reply by sweetppro** [reply]
@Oleg-Chashko Im currently looking at fixing Full Screen support.
Once I get that figured out, ill take a look at these
:)
